src/Entity/RememberDeviceToken.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Connect Holland Secure JWT package and distributed under the terms of the MIT License.
+ * Copyright (c) 2020 Connect Holland.
+ */
+
+namespace ConnectHolland\SecureJWTBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="secure_jwt_remember_device_token")
+ *
+ * @codeCoverageIgnore Trivial entity with only getters and setters.
+ */
+class RememberDeviceToken
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\Column(type="text")
```

Comment:
Do we really need text here? Wouldn't a string with a length suffice? Also for username